src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -47,8 +49,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // we're not relying on any extra magic inside UseServerSideBlazor, since it's
                 // important that people can set up these bits of middleware manually (e.g., to
                 // swap in UseAzureSignalR instead of UseSignalR).
-                subdirApp.UseSignalR(route => route.MapHub<ComponentsHub>(ComponentsHub.DefaultPath));
-                subdirApp.UseBlazor<BasicTestApp.Startup>();
+                subdirApp.UseRouting(routes => routes.MapComponentHub<Index>(ComponentHub.DefaultPath, selector: "root"));
```

Comment:
This has become inconsistent with the comment on lines 46-51. The intent expressed by the comment is that here we're demonstrating how to use Razor Components in combination with *manually configured* SignalR, to prove you could swap in AzureSignalR if you wanted.

Could L52 be changed back to having a call to `UseSignalR`, with the call that maps the components hub being separate?

Reply:
Can we not do this in test apps? Docs and samples are better for that.

Reply:
I'm not sure I agree. If this was only in a doc or sample, we wouldn't have even noticed we were losing support for this scenario. Test apps generally represent a wide range of use cases by design.

Reply:
I've updated it, but I can also argue that we should test the most common paths. Or if not then have tests for both.